settings.gradle

```diff
@@ -13,4 +13,4 @@ dependencyResolutionManagement {
     }
 }
 rootProject.name = "Tolkien Quiz App"
-include ':presentation'
+include ':presentation', ':domain', ':data'
```

Comment:
new modules have to be added to `include` - otherwise they won't be treated as modules, only "directories"